wa/starter/src/main/java/org/apache/syncope/wa/starter/config/SyncopeWAConfiguration.java

```diff
@@ -221,4 +237,14 @@ public KeymasterStart keymasterStart() {
     public KeymasterStop keymasterStop() {
         return new KeymasterStop(NetworkService.Type.WA);
     }
+
+    @Bean
+    public String version() {
+        return applicationContext.getEnvironment().getProperty("version");
+    }
+
+    @Bean
+    public String buildNumber() {
```

Comment:
This should not be needed.
Please also consider that it would require the `buildnumber-maven-plugin` to run.